docs/docsite/rst/administration/instances.rst

```diff
@@ -149,15 +151,15 @@ An instance has several attributes that may be configured:
 
 In the example diagram above, the configurations are as follows:
 
-+------------------+---------------+--------------------------+--------------+
-| instance name    | listener_port | peers_from_control_nodes | peers        |
-+==================+===============+==========================+==============+
-| execution node 1 | 27199         | true                     | []           |
-+------------------+---------------+--------------------------+--------------+
-| hop node         | 27199         | true                     | []           |
-+------------------+---------------+--------------------------+--------------+
-| execution node 2 | null          | false                    | ["hop node"] |
-+------------------+---------------+--------------------------+--------------+
++------------------+--------------------------------------------------+---------------+--------------------------+--------------+
+| instance type    | hostname                                         | listener_port | peers_from_control_nodes | peers        |
++==================+==================================================+===============+==========================+==============+
+| control plane    | awx-task-65d6d96987-mgn9j                        | 27199         | true                     | []           |
++------------------+--------------------------------------------------+---------------+--------------------------+--------------+
+| hop node         | awx-mesh-ingress-1                               | 27199         | true                     | []           |
++------------------+--------------------------------------------------+---------------+--------------------------+--------------+
+| execution node 2 | ec2-35-87-18-213.us-west-2.compute.amazonaws.com | null          | false                    | ["hop node"] |
++------------------+--------------------------------------------------+---------------+--------------------------+--------------+
 
 
```

Comment:
Add another table for the ingress mesh setup

Reply:
Table will go into the different topologies to describe the different peering scenarios.